storage/buckets/addBucketIamMember.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
```

Comment:
Please name the files with underscores instead of camelCase. Sorry, missed this in the other review.